src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const object = sourceString
+    .split(';')
+    .map((element) => {
+      return element.split(':').map((part) => part.trim());
+    })
+    .filter((array) => array.length !== 1)
+    .reduce((result, element) => {
```

Comment:
The variable name 'result' in the reduce callback is not descriptive. Since this represents the final object of CSS styles, a name like 'styleObject' or 'cssStyles' would be more informative.